packages/dht/src/helpers/EsmLoader.ts

```diff
@@ -0,0 +1,37 @@
+import { isBrowserEnvironment } from './browser/isBrowserEnvironment'
+import { waitForCondition } from '@streamr/utils'
+
+declare global {
+    // eslint-disable-next-line no-var, no-underscore-dangle
+    var _streamr_electron_test: boolean
+}
+
+export type NodeDataChannel = typeof import('node-datachannel', { with: { "resolution-mode": "import" } })
+
+// eslint-disable-next-line @typescript-eslint/no-extraneous-class
+export class EsmLoader {
```

Comment:
Is this something we can get rid of once (or if) we move to ESM packages ourselves? Maybe add a comment about this and a reference to ticket NET-963 related to moving the project into ESM.

Reply:
Yes, this EsmLoader is just a hack to make ESM modules work in our CommonJS codebase. Nothing like this will be needed if we move the whole project to ESM.